docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,43 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create a serializable resource instance
+serializable_resource = ActiveModel::SerializableResource.new(post, options)
+
+# Convert your resource into json
+model_json = serializable_resource.as_json
+```
+
+### Retrieving a Resource's Active Model Serializer
+
+If you want to retrieve a serializer for a specific resource, you can do the following:
```

Comment:
I'm not sure what the use case is, right now, for getting a **serializer** class *or* instance.  The only thing I can think of is to test that the serializer works as expected, which still requires and adapter...

If you can think of one, I think it would be good to add